docs/nx-cloud/reference/release-notes.md

```diff
@@ -147,6 +143,14 @@ NxCloud uses MongoDB internally as its data store. While we've always used Mongo
 While you can still upgrade to this new image even if you are on Mongo 4.2 (nothing will break), **we strongly recommend you upgrade your Database to Mongo 6.0 to make sure nothing breaks in the future.** [We wrote a full guide on how you can approach the upgrade here](https://github.com/nrwl/nx-cloud-helm/blob/main/MONGO-OPERATOR-GUIDE.md#upgrading-to-mongo-6).
 If you need assistance, please get in touch at [cloud-support@nrwl.io](mailto:cloud-support@nrwl.io).
 
+##### Migration from Community Edition to Enterprise
+
+A few weeks ago we announced our plans to sunset the Community Edition of NxCloud On-Prem to align with our new pricing plans. If you are on the Community Edition, please follow these steps to migrate:
```

Comment:
Don't use relative dates.  Instead, say something like:

On May 20th 2023, we announced...

Reply:
makes sense! updated.